#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_do_not_strip.dpatch by Daniel Kahn Gillmor <dkg@fifthhorseman.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: don't strip by default.  let dh_strip take care of it.

@DPATCH@
diff -urNad redir-2.2.1~/Makefile redir-2.2.1/Makefile
--- redir-2.2.1~/Makefile	2007-08-15 14:53:41.908911693 -0400
+++ redir-2.2.1/Makefile	2007-08-15 14:53:59.409909018 -0400
@@ -33,7 +33,7 @@
 OBJS = redir.o $(GETOPT_OBJS)
 
 CFLAGS = -O2 -Wall --pedantic $(STR_CFLAGS) $(WRAP_CFLAGS) $(EXTRA_CFLAGS)
-LDFLAGS = -s
+LDFLAGS = # -s
 
 # solaris, and others, may also need these libraries to link
 # also edit here if you're using the TCP wrappers code
